vm/tests/solutions/spec/alfresco/ports_spec.rb

```diff
@@ -19,7 +19,7 @@
 end
 
 describe port(5432) do
-  it { should be_listening.on('127.0.0.1') }
```

Comment:
Why are we opening the postgres database to all IPs now?

Reply:
Fixed. Not opening to all IPs right now.